src/main/java/randoop/execution/RunCommand.java

```diff
@@ -0,0 +1,126 @@
+package randoop.execution;
+
+import java.io.File;
+import java.io.IOException;
+import java.util.List;
+import plume.TimeLimitProcess;
+import randoop.util.StreamUtils;
+
+/**
+ * Class providing the {@link #run(List, File, long)} method to run a command in a separate process
+ * with a timeout.
+ */
+public class RunCommand {
```

Comment:
Just a note for the future: we discussed this and decided it could be done later.